<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>main</class>
 <widget class="QMainWindow" name="main">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>446</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>main</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>261</width>
      <height>231</height>
     </rect>
    </property>
    <property name="accessibleName">
     <string/>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="btnUser">
       <property name="text">
        <string>USER</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPelanggan">
       <property name="text">
        <string>PELANGGAN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnAlat">
       <property name="text">
        <string>ALAT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPeminjaman">
       <property name="text">
        <string>PEMINJAMAN</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPembayaran">
       <property name="text">
        <string>PEMBAYARAN</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>401</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>APLIKASI JASA PEMINJAMAN ALAT MUSIK</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>446</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuduwqegfhen">
    <property name="title">
     <string>HALAMAN UTAMA</string>
    </property>
    <addaction name="actionkkjckasdv"/>
    <addaction name="actionFORM_PELANGGAN"/>
    <addaction name="actionFORM_ALAT"/>
    <addaction name="actionFORM_PEMINJAMAN"/>
    <addaction name="actionFORM_PEMBAYARAN"/>
   </widget>
   <addaction name="menuduwqegfhen"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionkkjckasdv">
   <property name="text">
    <string>FORM USER</string>
   </property>
  </action>
  <action name="actionFORM_PELANGGAN">
   <property name="text">
    <string>FORM PELANGGAN</string>
   </property>
  </action>
  <action name="actionFORM_ALAT">
   <property name="text">
    <string>FORM ALAT</string>
   </property>
  </action>
  <action name="actionFORM_PEMINJAMAN">
   <property name="text">
    <string>FORM PEMINJAMAN</string>
   </property>
  </action>
  <action name="actionFORM_PEMBAYARAN">
   <property name="text">
    <string>FORM PEMBAYARAN</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
